#' Genome annotation file.
#'
#' A dataset containing the annotation file WITHOUT the column name. The example file is Maize genome 
#' annotation, and the version is AGPv2.IMPORTANT, the annotation file could be read with header=FALSE.
#' The variables are as follows:
#'
#' @format:
#' \describe{
#'   \item{the first column}{The name of the sequence. Commonly, this is the chromosome ID or contig ID}
#'   \item{the second column}{The source column should be a unique label indicating whether the annotation is protein-coding or something else.}
#'   \item{the third column}{The following feature types are required: 'CDS', 'start_codon', 'stop_codon'. The features '5UTR', '3UTR', 'inter', 'inter_CNS', 'intron_CNS' and 'exon' are optional.}
#'   \item{the fourth column}{Integer start coordinates of the feature relative to the beginning of the sequence named}
#'   \item{the fifth co0umn}{Integer end coordinates of the feature relative to the beginning of the sequence named}
#'   \item{the sixth column}{The score field indicates a degree of confidence in the feature's existence and coordinates.}
#'   \item{the seventh column}{The strand indicates the annotation is located on forward or reverse strand.}
#'   \item{the eigth column}{frame}
#'   \item{the ninth column}{attributes}
#' }
"gtf"

#' GWAS result.
#'
#' A dataset containing the association file. The variables are as follows:
#'
#' @format:
#' \describe{
#'   \item{Marker}{molecular marker name}
#'   \item{Locus}{the chromosome of the marker}
#'   \item{Site}{the position of the marker}
#'   \item{p}{p value of the marker.}
#' }
"association"

#' Genotype file.
#'
#' A dataset containing the genotype file. The variables are as follows:
#'
#' @format:
#' \describe{
#'   \item{rs}{contains the SNP identifier.}
#'   \item{alleles}{contains SNP alleles according to NCBI database dbSNP.}
#'   \item{chrom}{contains the chromosome that the SNP was mapped.}
#'   \item{pos}{contains the respective position of this SNP on chromosome.}
#'   \item{strand}{contains the orientation of the SNP in the DNA strand. Thus, SNPs could be in the forward (+) or in the reverse (-) orientation relative to the reference genome.}
#'   \item{assembly}{contains the version of reference sequence assembly (from NCBI).}
#'   \item{center}{contains the name of genotyping center that produced the genotypes.}
#'   \item{protLSID}{contains the identifier for HapMap protocol.}
#'   \item{assayLSID}{contain the identifier HapMap assay used for genotyping.}
#'   \item{panelLSID}{contains the identifier for panel of individuals genotyped.}
#'   \item{QCcode}{contains the quality control for all entries.} subsequently, the list of sample names.
#' }
"hapmap_am368"

#' Names of genotype name file.
#'
#' A dataset containing the name of genotype file. The variables are as follows:
#'
#' @format:
#' \describe{
#'   \item{rs}{contains the SNP identifier.}
#'   \item{chrom}{contains the chromosome that the SNP was mapped.}
#'   \item{pos}{contains the respective position of this SNP on chromosome.}
#'   \item{shape}{contains the respective shape of this SNP on chromosome.}
#'   \item{colour}{contains the respective colour of this SNP on chromosome.}
#'   \item{fill}{contains the respective fill of this SNP on chromosome.}
#'   \item{size}{contains the respective size of this SNP on chromosome.}
#' }
"marker2highlight"

#' Names of genotype name file.
#'
#' A dataset containing the name of genotype file. The variables are as follows:
#'
#' @format:
#' \describe{
#'   \item{rs}{contains the SNP identifier.}
#'   \item{chrom}{contains the chromosome that the SNP was mapped.}
#'   \item{pos}{contains the respective position of this SNP on chromosome.}
#' }
"marker2link"

#' Genotype file.
#'
#' A dataset containing the genotype file. The variables see hapmap_am368
"hapmap2"

#' Genotype file.
#'
#' A dataset containing the genotype file generated by resequence ZmVpp1. The variables see hapmap_am368
"zmvpp1_hapmap"

#' GWAS result.
#'
#' A dataset containing the association file from the resequenced ZmVpp1. The variables see association:
"zmvpp1_association"
